src/src.pro

```diff
@@ -86,6 +93,11 @@ HEADERS += \
     udisks2monitor_p.h \
     userinfo_p.h
 
+CONFIG(DEVELOPER_MODE_ENABLED) {
```

Comment:
So there's both CONFIG(disableDevmode) and CONFIG(DEVELOPER_MODE_ENABLED)?

Reply:
fixed